src/app/app.component.ts

```diff
@@ -11,8 +11,8 @@ import { NavigationEnd, Router } from '@angular/router';
 })
 export class AppComponent implements OnInit {
     constructor(
-        private angulartics2GoogleTagManager: Angulartics2GoogleAnalytics,
-        private angulartics2Intercom: Angulartics2Intercom,
+        // tslint:disable:no-unused-variable
+        private angulartics2GoogleTagManager: Angulartics2GoogleAnalytics, private angulartics2Intercom: Angulartics2Intercom,
```

Comment:
Two things:

- I think the `tslint:disable:...` comment disables TSLint for _the rest of the file_. You should probably use `tslint:disable-next-line` if possible
- I think we usually do each dependency on its own line.